nixos/modules/installer/cd-dvd/installation-cd-graphical-calamares-base.nix

```diff
@@ -0,0 +1,66 @@
+# This module contains the basic configuration for building a graphical NixOS
+# installation CD.
+
+{ lib, pkgs, ... }:
+
+with lib;
+let
+  calamares-nixos-autostart = (pkgs.makeAutostartItem { name = "calamares"; package = pkgs.calamares-nixos; });
+in
+{
+  imports = [ ./installation-cd-base.nix ];
+
+  # Whitelist wheel users to do anything
+  # This is useful for things like pkexec
+  #
+  # WARNING: this is dangerous for systems
+  # outside the installation-cd and shouldn't
+  # be used anywhere else.
+  security.polkit.extraConfig = ''
+    polkit.addRule(function(action, subject) {
+      if (subject.isInGroup("wheel")) {
+        return polkit.Result.YES;
+      }
+    });
+  '';
+
+  services.xserver.enable = true;
+
+  # Provide networkmanager for easy wireless configuration.
+  networking.networkmanager.enable = true;
+  networking.wireless.enable = mkForce false;
```

Comment:
I removed this file entirely, but in [installation-cd-graphical-base.nix](https://github.com/vlinkz/nixpkgs/blob/df0d9e6da95b3768eadfa826b44db92610039c31/nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix#L29), `mkForce` is required to avoid the following error:
```
error: The option `networking.wireless.enable' has conflicting definition values:
       - In `nixpkgs/nixos/modules/profiles/installation-device.nix': true
       - In `nixpkgs/nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix': false
```